cli/exchange/agbot.go

```diff
@@ -147,5 +149,7 @@ func AgbotRemoveBusinessPolicy(org, userPw, agbot, PolOrg string) {
 	cliutils.SetWhetherUsingApiKey(userPw)
 	org, agbot = cliutils.TrimOrg(org, agbot)
 	polId := formServicedObjectId(PolOrg, "*", PolOrg)
+	i18n.GetMessagePrinter().Printf("Removing Business policy org %s' from agbot '%s' ...", PolOrg, agbot)
```

Comment:
Removing Business policy org %s' to agbot '%s'. The agbot is no longer looking for nodes that are compatible with this policy."